ElectronClient/app/gui/MainScreen.jsx

```diff
@@ -115,6 +115,16 @@ class MainScreenComponent extends React.Component {
 								bridge().showErrorMessageBox(error.message);
 							}
 
+							if (command.name === 'newSubNotebook') {
+								let subFolder = null;
+								try  {
+									subFolder = await Folder.moveToFolder(folder.id, command.activeFolderId);
+								} catch (error) {
+									bridge().showErrorMessageBox(error.message);
+								}
+								folder = subFolder;
+							}
+
```

Comment:
I think this can be simplified to `if (command.name === 'newSubNotebook') folder = await Folder.moveToFolder(folder.id, command.activeFolderId);` and moving this under the try block above. That way we don't repeat try/catch logic and error handling.

Reply:
ok, makes sense.